Projects/Server/MessageType.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
```

Comment:
Add copyright. If you are using Rider, you can import rider-settings.zip and do `copyright` and select it from the autocomplete dropdown to generate it.

Reply:
Added!